readthedocs/source/index.rst

```diff
@@ -176,4 +176,28 @@ Image address:
 
 .. code-block::
 
-  https://llm-assets.readthedocs.io/en/latest/_images/llm-finetune-lora-cpu-k8s.png
\ No newline at end of file
+  https://llm-assets.readthedocs.io/en/latest/_images/llm-finetune-lora-cpu-k8s.png
+
+
+qwen-vl-example-input.jpg
+---------------------------------
+.. figure:: ../../assets/qwen-vl-example/qwen-vl-example-input.jpg
```

Comment:
let's change the folder name to llm-example :) we could add more example picture in the same folder later